captum/optim/models/_common.py

```diff
@@ -202,7 +202,10 @@ class SkipLayer(torch.nn.Module):
     This layer is made to take the place of nonlinear activation layers like ReLU.
     """
 
-    def forward(self, x: torch.Tensor) -> torch.Tensor:
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
why are the args and kwargs passed if they are not used ? Are there any real scenarios that we need it ? Is this because we replace ReLU with SkipLayer ?
I could imagine, for example, that x is a tuple of tensors and we would need to return that tuple.

Reply:
I made the change because users may have models using `activ(inplace=False)` or `activ(False)` where `activ = torch.nn.ReLU`. This is same way that `torch.nn.Identity` works: https://pytorch.org/docs/stable/generated/torch.nn.Identity.html

I'll add the type hint for tuples of tensors.